### DISCUSSION TITLE
Your active account is not authenticated.

### DISCUSSION THREAD

**Discussion by daFiter** [EsriPS/arcgis-assistant-feedback#54] [Q&A]
For some reason I am getting that my account is not authenticated. I am setting up a new 10.9.1 server and I am not sure why I am getting this error.

**Comment by jpeterson** [comment]
Hey @daFiter, could you share some additional details about what you're seeing? Any screenshots or error messages from your browser's console window? Thanks!

**Comment by daFiter** [comment]
![Screenshot 2022-10-18 055740](https://user-images.githubusercontent.com/10452281/196435996-fb397b33-4d80-487c-8745-c606e4f878b1.jpg)

![Screenshot 2022-10-18 055857](https://user-images.githubusercontent.com/10452281/196436265-325203d1-ae09-4c31-a448-465ba8832361.jpg)

**Reply by AlexWilkinsonEagle** [reply]
Hi there, I had this error and managed to resolve it by adding the URL for the ArcGIS Assistant to the 'Allow Origins' section of the Security settings for Portal.

However, make sure that the URL **doesn't** have a forward slash at the end or it won't work. So ensure the URL is added just like below:

![image](https://github.com/EsriPS/arcgis-assistant-feedback/assets/101836671/3a4e75e4-b21a-4bfd-a71a-bb09965185cb)

**Comment by karengault** [comment]
Has anyone else found a resolution to this?  This behavior just started for us, where just last week it worked.  Exact same error message in the dev tools console.

**Comment by slibby** [comment]
This looks to me like it may be an issue with the browser trusting your Portal's SSL/TLS certificate. Can you try adding the TLS certificate to your trusted certificates and re-connecting? I believe the "ERR_CERT_AUTHORITY_INVALID" (thrown by a cross-origin request to your Portal) is what's causing the issue.

**Reply by karengault** [reply]
I'm not actually seeing that error; only the "Error getting User Session (completeOAuth). Error reading property....." etc.

**Reply by slibby** [reply]
OK, I had gone off of your comment of the exact same error message, but maybe it was just that one-- Do you see any failed requests in the Network tab, or any that are returning a JSON error?

**Reply by karengault** [reply]
We're actually starting to see CORS errors for a bunch of stuff trying to access this portal, so we think it's related.  Unsure why; the origins are listed in both Trusted Servers and Allowed Origins on the portal settings, and we've made sure on the server that the Allowed Origins is set to *.

**Reply by slibby** [reply]
Do you see assistant.esri-ps.com listed in the Access-Control-Allow-Origin header response on requests sent from the Assistant UI to your Portal?
